data/ka'het ships.txt

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2019 by Tommaso Becca
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+ship "Maeri'het"
+	plural "Maeri'heta"
+	sprite "ship/maeri'het"
+	"never disabled"
+	attributes
+		category "Light Warship"
+		"cost" 13800000
+		"shields" 17600
+		"hull" 23100
+		"required crew" 1
+		"bunks" 1
+		"mass" 390
+		"drag" 4.1
+		"heat dissipation" .4
+		"fuel capacity" 800
+		"ramscoop" 0.58
+		"cargo space" 43
+		"outfit space" 273
+		"weapon capacity" 184
+		"energy capacity" 10000
+		"energy generation" 28.3
+		"heat generation" 32
+		"hull repair" 0.3
+		"shield heat" 4.9
+		"cloak" .03
+		"cloaking energy" 9.4
+		"cloaking fuel" .08
+		"thrust" 39.2
+		"thrusting energy" 6.17
+		"thrusting heat" 1.8
+		"flare sprite" "effect/remnant flare/small"
+			"frame rate" 5
+		"flare sprite" "effect/remnant flare/small"
+			"frame rate" 5
+		"flare sound" "plasma small"
+		"turn" 986.4
+		"turning energy" 3.14
+		"turning heat" 1.4
+		weapon
+			"blast radius" 180
+			"shield damage" 720
+			"hull damage" 360
+			"hit force" 1080
+	outfits
+		"Hyperdrive"
+		"Ka'het Small Cooling" 
+		"Ka'het Large Cooling"
+		"Ka'het Blaster Turret" 2
+		"Ka'het Laser Turret" 2
+		"Ka'het Laser" 2
+		"Ka'het Shield" 2
+
+	engine -20 66 0.6
+	engine -26.5 66 0.8
+	engine 20 66 0.6
+	engine 26.5 66 0.8
+	gun -9 -74 "Ka'het Laser"
+	gun 9 -74 "Ka'het Laser"
+	turret -13.5 -29 "Ka'het Blaster Turret"
+	turret -14 -12.5 "Ka'het Laser Turret"
+	turret 13.5 -29 "Ka'het Blaster Turret"
+	turret 14 -12.5 "Ka'het Laser Turret"
+	explode "tiny explosion" 5
+	explode "small explosion" 20
+	explode "medium explosion" 25
+	explode "large explosion" 15
+	"final explode" "final explosion medium"
+	description "The Maeri is the smallest Het still created by the Builders' stations, a prime example of the genetic technology they once possessed. Its small size is compensated by the fact that this specie of Het can effectively cloak, hiding its location from most types of sensors."
```

Comment:
description "The Maeri is the smallest Het vessel created by the Builders' stations, a prime example of the genetic technology they once possessed. Its small size is compensated by the fact that it has cloaking technology, hiding its location from most types of sensors."